website/docs/reference/commands/dbt-environment.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
```

Comment:
@cesher one last question - what is the difference between local config and dbt cloud config in this context? i think i understand the difference in the 'git' context but wanted to clarify this for users. 

is it:
- local config -- details about their local environment setup?
- cloud config - dbt cloud account and connection details

Reply:
going to merge pr and address this when clarified in the shipped channel